chiselFrontend/src/main/scala/chisel3/core/Aggregate.scala

```diff
@@ -408,33 +415,19 @@ class Bundle extends Aggregate {
           constructor.newInstance(_parent.get).asInstanceOf[this.type]
         } catch {
           case _: java.lang.reflect.InvocationTargetException | _: java.lang.IllegalArgumentException =>
-            Builder.error(s"Parameterized Bundle ${this.getClass} needs cloneType method. You are probably using " +
-              "an anonymous Bundle object that captures external state and hence is un-cloneTypeable")
+            Builder.error(s"Parameterized Record ${this.getClass} needs cloneType method. You are probably using " +
+              "an anonymous Record object that captures external state and hence is un-cloneTypeable")
             this
         }
       case _: java.lang.reflect.InvocationTargetException | _: java.lang.IllegalArgumentException =>
-        Builder.error(s"Parameterized Bundle ${this.getClass} needs cloneType method")
+        Builder.error(s"Parameterized Record ${this.getClass} needs cloneType method")
         this
     }
   }
-
-  /** Default "pretty-print" implementation
-    * Analogous to printing a Map
-    * Results in "Bundle(elt0.name -> elt0.value, ...)"
-    */
-  def toPrintable: Printable = {
-    val elts =
-      if (elements.isEmpty) List.empty[Printable]
-      else {
-        elements.toList.reverse flatMap { case (name, data) =>
-          List(PString(s"$name -> "), data.toPrintable, PString(", "))
-        } dropRight 1 // Remove trailing ", "
-      }
-    PString("Bundle(") + Printables(elts) + PString(")")
-  }
 }
 
 private[core] object Bundle {
```

Comment:
I don't think they are used, but I think removing them is a separate PR since it kind of goes along with modifying the Namespace